test/docker-compose.integration-tests.yml

```diff
@@ -160,14 +71,24 @@ services:
       - GRAPL_AWS_ENDPOINT
       - GRAPL_HTTP_FRONTEND_PORT
       - GRAPL_LOG_LEVEL=${GRAPL_LOG_LEVEL:-INFO}
+      - GRAPL_NOTEBOOK_INSTANCE="local-grapl-Notebook"
       - GRAPL_TEST_USER_NAME
+      - HITCACHE_ADDR=${REDIS_HOST}
+      - HITCACHE_PORT=${REDIS_PORT}
       - IS_LOCAL=True
+      - IS_RETRY=False
+      - MESSAGECACHE_ADDR=${REDIS_HOST}
+      - MESSAGECACHE_PORT=${REDIS_PORT}
       - MG_ALPHAS
+      - UX_BUCKET_URL="ux_bucket_url"
       - VSC_DEBUGGER_PORT=${VSC_DEBUGGER_PORT_FOR_GRAPHQL_ENDPOINT_TESTS}
     ports:
       - ${VSC_DEBUGGER_PORT_FOR_GRAPHQL_ENDPOINT_TESTS}:${VSC_DEBUGGER_PORT_FOR_GRAPHQL_ENDPOINT_TESTS}
     volumes:
       - *read-only-pulumi-mnt
+      # mount the grapl repo rw
+      - "../:/mnt/grapl"
```

Comment:
personally, i prefer the long form for volumes (because I always need to look up if this is a volume or a mount) but whatever

Reply:
I do to, but as far as I could tell there's no way to express "mount `..` at `/mnt/grapl`" in the long form

Reply:
fixed